eradiate/scenes/illumination/_constant.py

```diff
@@ -21,13 +23,13 @@ class ConstantIllumination(Illumination):
         doc="Emitted radiance spectrum. Must be a radiance spectrum "
         "(in W/m^2/sr/nm or compatible units).",
         type="float or :class:`~eradiate.scenes.spectra.Spectrum`",
-        default="1.0 cdu[radiance]",
+        default="1.0 ucc[radiance]",
```

Comment:
Does `ucc['radiance']` work already?

Reply:
I don't think the Pinttrs update has been propagated to Eradiate yet; but it should work with the latest version if I'm not mistaken.